### PR TITLE
Remove simulation classes

### PR DESCRIPTION
**Pull request by francisco-dlp**
Remove simulation classes as discussed in #1103.

### CODE REVIEW

File: hyperspy/io.py [hunk 1]
```diff
@@ -326,19 +306,13 @@ def assign_signal_subclass(record_by="",
...
     signals = hyperspy.misc.utils.find_subclasses(hyperspy.signals, BaseSignal)
 
-    if signal_origin == "experiment":
-        signal_origin = ""
-
     preselection = [s for s in
                     [s for s in signals.values()
                      if record_by == s._record_by]
-                    if signal_origin == s._signal_origin]
+                    ]
```
Comment (to266) [inline hyperspy/io.py]: This preselection is now meaningless - just passes all `signals.values()`

**Comment by to266** [comment]
Oops, forgot I commented... It's not important though, and it will have to be updated with lazy signals at some point

**Comment by francisco-dlp** [comment]
No worries, that function has been refurbished in #1058

**Comment by francisco-dlp** [comment]
appveyor is randomly failing like that lately.

**Comment by francisco-dlp** [comment]
I've triggered another build, but if it fails I don't think that there is anything to worry about as this PR is very straightforward.

**Pull request merged**